src/pytest_cov/plugin.py

```diff
@@ -333,6 +338,8 @@ def pytest_runtestloop(self, session):
             # it for unit tests that don't need it
             from coverage.misc import CoverageException
 
+            tr = TerminalReporter(session.config, self.cov_report)
+            tr.write_sep('=', 'coverage report')
```

Comment:
Why does this need to be added to the `StringIO` at all? If you instead output the separator in `pytest_terminal_summary`, couldn't you use the `terminalreporter` there which is used to output the report, instead of having to create a new one?